src/as_execution/execution.rs

```diff
@@ -108,7 +108,7 @@ impl ASContextModule {
                         init_cost: 0,
                     });
                 }
-                let ret = if let Some(offset) = value.get(0) {
+                let ret = if let Some(offset) = value.first() {
```

Comment:
If you feel this can make the code more readable, you can use the let-else syntax which was recently stabilized:

```rust
let Some(offset) = value.first() else {
    return Vec::new();
};
// construct the ret variable here
```

Though this is probably out of scope for this PR. Feel free to go ahead and resolve this conversation without comment.

Reply:
...started a discussion here: https://github.com/massalabs/massa/discussions/3494

Reply:
Don't think it hurts applying this change here. In any case I approved.